src/harness/testcases/configs/test_WINNF_FT_S_FDB_1/default.config

```diff
@@ -0,0 +1,132 @@
+{
```

Comment:
Please remove the config files and the unrelated .json files.

Reply:
removed